charts/agent/values.yaml

```
@@ -20,7 +20,12 @@ observe:
 
 cluster:
   # name given to your cluster
+  # popuplates to k8s.cluster.name per https://opentelemetry.io/docs/specs/semconv/resource/k8s/#cluster
   name: observe-agent-monitored-cluster
+  deploymentEnvironment:
+    # deployment environment cluster runs in (e.g. testing, production, etc)
```

If you add `--` then the comment will be added to the readme; ie:
```suggestion
    # -- deployment environment cluster runs in (e.g. testing, production, etc)
```

---

Neat!